rest/endpoints/auth/authentication.yml

```diff
@@ -36,45 +36,22 @@ paths:
             schema:
               type: object
               required:
-              - authentication_mechanism
+              - grant_type
               - credentials
               properties:
-                authentication_mechanism:
+                grant_type:
                   type: string
                   example: jwt
                   description: Any of the (case-insensitive) strings [entone|password|token|refresh]
                   enum: [entone|password|token|refresh]
```

Comment:
Let's update these options too. For now only `stb_pwless` and `refresh_token` are supported